### DISCUSSION TITLE
Is it possible to use nchat to download all media to Downloads folder?

### DISCUSSION THREAD

**Discussion by fachinformatiker** [d99kris/nchat#144] [Q&A]
Maybe I can't find the option but I want to automatically download all new media (images, voice memos, etc.) into my download folder (provided via config)
I enabled the prefetch but it doesn't download any files.

**Comment by d99kris** [comment]
Hi @fachinformatiker - so there's no such function in nchat today. What **prefetch** does is to download media to nchat's cache folders, for example for Telegram:
```~/.nchat/profiles/Telegram_+nnnnn/tdlib/documents```
```~/.nchat/profiles/Telegram_+nnnnn/tdlib/videos```
(separate folder for different types)
or in the case of WhatsApp:
```~/.nchat/profiles/WhatsAppMd_+nnnnn/tmp```

This is so that viewing/opening a file in nchat can be done instantly. There's currently no plan to add support for auto-saving/downloading such files to a user-specified folder.
